### DISCUSSION TITLE
Add color to the node

### DISCUSSION THREAD

**Discussion by BrittoAlbin** [joyceworks/flowchart-react#26] [Q&A] [answered]
How to add color to the nodes dynamically

**Comment by iamppz** [comment]
Colors are currently not customizable, but I will add support for custom svg attributes (including background and border colors) in the next few days.

**Reply by BrittoAlbin** [reply]
Thanks for the reply. It would be better if you add one more property for color in NodeData props and provide flexibility to pass color.I think currently it is hard coded as white in nodes.Thanks.

**Comment by iamppz** [comment, marked as answer]
Please upgrade flowchart-react to 3.8.1, use `containerProps` & `textProps` to change node color:
``` typescript
const [nodes, setNodes] = useState<NodeData[]>([
  ...
  {
    x: 400,
    y: 240,
    id: 3,
    title: "Joyce",
    type: "operation",
    containerProps: {fill: "blue", stroke: "red"},
    textProps: {fill: "yellow"}
  },
]);
```

**Reply by BrittoAlbin** [reply]
Great work.Thanks.
